controllers/kafkamanagement/kafkauser_controller.go

```diff
@@ -79,291 +75,318 @@ func (r *KafkaUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 		l.Error(err, "Unable to fetch Kafka User", "request", req)
 		return models.ReconcileRequeue, nil
```

Comment:
Let's add an event if Kafka user is not found

Reply:
done